hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -83,5 +83,7 @@ rm -rf out/buildroot
 # -R: recursive. strangely, this is not the default for sync.
 gsutil -m rsync -dJR out "gs://${bucket}/${ghprbPullId}"
 
-# Copy artifacts to known mirror location
-gsutil cp -R "gs://${bucket}/${ghprbPullId}" "gs://${bucket_mirror}"
+if [[ "${ghprbPullId}" -eq "master" ]] then
```

Comment:
Does this need an `;` between the conditional then `then`? Only asking because the rest of them have it.

```suggestion
if [[ "${ghprbPullId}" -eq "master" ]]; then
```

Reply:
good call, just a mistake on my part